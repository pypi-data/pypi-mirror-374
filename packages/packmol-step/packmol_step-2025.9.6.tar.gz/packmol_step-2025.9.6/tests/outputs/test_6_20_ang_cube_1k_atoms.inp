tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside cube 0.0 0.0 0.0 20.0000
   number 6
end structure
structure input_2.pdb
   inside cube 0.0 0.0 0.0 20.0000
   number 291
end structure
