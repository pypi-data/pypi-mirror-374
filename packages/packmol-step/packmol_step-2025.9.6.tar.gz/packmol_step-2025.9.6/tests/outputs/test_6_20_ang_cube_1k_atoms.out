    Will create a cubic region containing the following molecules:

        +-------------+------------------+---------+
        |  Component  | Structure        |   Ratio |
        |-------------+------------------+---------|
        |    fluid    | c1ccccc1c2ccccc2 |       1 |
        |    fluid    | O                |      50 |
        +-------------+------------------+---------+

    The dimensions of the region will be given explicitly by the edge length
    20.0 Å. The number of molecules of the fluid will be obtained by rounding
    1000 atoms to give a whole number of molecules with the requested ratios.

Created a cubic region 20.00 Å on a side with 297 fluid molecules

    +-------------+------------------+---------------+----------+------------+
    |  Component  | Structure        |   Requested % |   Number |   Actual % |
    |-------------+------------------+---------------+----------+------------|
    |    fluid    | c1ccccc1c2ccccc2 |         1.961 |        6 |       2.02 |
    |    fluid    | O                |        98.039 |      291 |      97.98 |
    +-------------+------------------+---------------+----------+------------+

There are a total of 1005 atoms in the cell giving a density of 1.2802 g/ml.